include/seqan3/range/views/interleave.hpp

```diff
@@ -61,10 +61,12 @@ class view_interleave : public std::ranges::view_interface<view_interleave<urng_
     //!\brief The range to be inserted into urange.
     inserted_rng_t inserted_range;
 
-public:
     /*!\name Associated types
+     *!\brief These associated types are needed in random_access_iterator.
      * \{
      */
+    //!\brief This resolves to range_type::size_type as the underlying range is guaranteed to be Sized.
+    using size_type         = size_type_t<urng_t>;
```

Comment:
Does this work?
```suggestion
    using size_type         = std::ranges::range_size_t<urng_t>;
```

Reply:
Yes, it does. This was part of https://github.com/seqan/seqan3/pull/1777, fixed with the rebase :)